themes/edx.org/lms/templates/dashboard.html

```diff
@@ -146,7 +146,7 @@
       <h2 class="header-courses">${_("My Courses")}</h2>
     </header>
     % if len(course_entitlements + course_enrollments) > 0:
-        <ul class="listing-courses">
+        <ul class="listing-courses" data-programs=${programs_data}>
```

Comment:
Maybe leave a comment of the form:
```
TODO: LEARNER-XXX: Clean up if dashboard program data is not productized.
```
Or, in LEARNER-XXX, you could add an AC to find and delete this.

Reply:
added a comment to cover this and the other code above, also added subtask to the ticket